models/model.js

```diff
@@ -82,5 +82,32 @@ exports.getArticles = async() => {
 
     catch(err) {
         
+    }
+}
+
+exports.getAllComments = async (article_id) => {
+    try {
+        const query = await connection.query(`
+
+        SELECT * FROM comments WHERE article_id = ${article_id};
+
```

Comment:
I'd consider refactoring this to add in something to arrange the order 👀  rather than the code on lines 102 - 103